gradle/libs.versions.toml

```diff
@@ -94,7 +95,10 @@ android-svg = { module = "com.caverock:androidsvg-aar", version.ref = "android-s
 coil-compose = { module = "io.coil-kt.coil3:coil-compose", version.ref = "coil" }
 coil-network-ktor3 = { module = "io.coil-kt.coil3:coil-network-ktor3", version.ref = "coil" }
 
+aboutlibraries-compose = { module = "com.mikepenz:aboutlibraries-compose", version.ref = "aboutlibraries" }
+
```

Comment:
Licenses list is pretty simple IMHO, and can be easily implemented in the app without 3rd party libraries, to better adhere to our design principles.
Can we use the library only to collect license information, but render it ourselves?

Reply:
Yes! It can give us just the data itself and then we can render it arbitrarily.